src/ResourceManager/RecoveryServices/ChangeLog.md

```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Current Release
+* Fixed bug - Make vault credential cmdlet test compulsory for Check-In.
```

Comment:
@dragonfly91 if the only changes in this module were test changes, then you can go ahead and revert the changes made to this change log. There shouldn't be a need to ship another iteration of `RecoveryServices` if there were no changes impacting customers.